Presentation.Web/Tests/it-system/tabs/LocalAdmin.Reference.System.e2e.spec.ts

```diff
@@ -0,0 +1,89 @@
+﻿import login = require("../../Helpers/LoginHelper");
+import HomePage = require("../../PageObjects/it-system/tabs/ItSystemReference.po");
+import WaitUpTo = require("../../Utility/WaitTimers");
+import ReferenceHelper = require("../../Helpers/ReferenceHelper");
+import Constants = require("../../Utility/Constants");
+import TestFixtureWrapper = require("../../Utility/TestFixtureWrapper");
+
+var loginHelper = new login();
+var homePage = new HomePage();
+var waitUpTo = new WaitUpTo();
+var refHelper = new ReferenceHelper();
+var consts = new Constants();
+var testFixture = new TestFixtureWrapper();
+
+var headerButtons = homePage.kendoToolbarWrapper.headerButtons();
+var inputFields = homePage.kendoToolbarWrapper.inputFields();
+var colObjects = homePage.kendoToolbarWrapper.columnObjects();
+
+describe("Local admin can add and remove reference",
+    () => {
+        beforeAll(() => {
+            loginHelper.loginAsLocalAdmin();
+        });
+
+        beforeEach(() => {
+            homePage.getPage();
+        });
+
+        afterAll(() => {
+            testFixture.cleanupState();
+        });
+
+        it("",
```

Comment:
it("Can add and remove references") - Bedre end describe beskrivelse som bør være generel

Reply:
Rettet tekst til "Can add and remove reference"